### ISSUE TITLE
Cannot log in after updated to the 1.39.1

### ISSUE THREAD

**Issue by MFYDev** [umami-software/umami#1583]
Hello, I cannot log in anymore after the update.

I got 500 error when post the credentials to the /auth/login

I am using Nginx and did not have that issue before

**Comment by MFYDev** [comment]
Below here is my Nginx configuration:

location ^~ /
{
    proxy_pass http://t.mfy.world:9850;
    proxy_set_header Host t.mfy.world;
    proxy_set_header X-Real-IP $remote_addr;
    proxy_set_header X-Forwarded-For $proxy_add_x_forwarded_for;
    proxy_set_header REMOTE-HOST $remote_addr;
    proxy_set_header Upgrade $http_upgrade;
    proxy_set_header Connection "upgrade";
}
#PROXY-END/

**Comment by benscobie** [comment]
Me too. Here's the error in my log:
4 migrations found in prisma/migrations
✓ Database is up to date.
$ node scripts/update-tracker.js
$ node server.js
Listening on port 3000
PrismaClientKnownRequestError: 
Invalid `prisma.account.findUnique()` invocation:
The column `account.account_uuid` does not exist in the current database.

**Comment by nicolasluckie** [comment]
I had similar error after updating to v1.39.1 - I was able to fix and restore all my data after executing the following commands on the postgres database:

## 1) `CREATE EXTENSION pgcrypto;`

Learned the hard way that you need to load the *pgcrypto* extension in the current database/schema first.

You otherwise get the following error:
```
[53] ERROR:  function gen_random_uuid() does not exist at character 33
[53] HINT:  No function matches the given name and argument types. You might need to add explicit type casts.
```

## 2) Copy/Paste

If using docker-compose: `docker exec -it <umami_db_container_name> psql -U umami -W umami`

Enter your db password and copy/paste the following:

```
-- AlterTable
ALTER TABLE "account" ADD COLUMN "account_uuid" UUID NULL;

-- Backfill UUID
UPDATE "account" SET account_uuid = gen_random_uuid();

-- AlterTable
ALTER TABLE "account" ALTER COLUMN "account_uuid" SET NOT NULL;

-- CreateIndex
CREATE UNIQUE INDEX "account_account_uuid_key" ON "account"("account_uuid");

-- AlterTable
ALTER TABLE "event" ADD COLUMN "event_uuid" UUID NULL;

-- Backfill UUID
UPDATE "event" SET event_uuid = gen_random_uuid();

-- AlterTable
ALTER TABLE "event" ALTER COLUMN "event_uuid" SET NOT NULL;

-- CreateIndex
CREATE UNIQUE INDEX "event_event_uuid_key" ON "event"("event_uuid");

-- CreateIndex
CREATE INDEX "account_account_uuid_idx" ON "account"("account_uuid");

-- CreateIndex
CREATE INDEX "session_session_uuid_idx" ON "session"("session_uuid");

-- CreateIndex
CREATE INDEX "website_website_uuid_idx" ON "website"("website_uuid");

-- CreateIndex
CREATE INDEX "event_event_uuid_idx" ON "event"("event_uuid");

```

Hope this helps! -Nic

**Comment by benscobie** [comment]
Thanks @nicolasluckie - I checked the _prisma_migrations table and saw the same error. I just ran
`CREATE EXTENSION IF NOT EXISTS pgcrypto;`
`delete from _prisma_migrations where migration_name = '04_add_uuid';`

And restarted Umami to re-apply.

**Comment by briancao** [comment]
@nicolasluckie Thank you for the quick solution!

It looks like gen_random_uuid() was added in postgres v13+ natively (https://pgpedia.info/g/gen_random_uuid-function.html) and I should have taken versions into consideration.

**Comment by mskian** [comment]
tried this work for me but issue on real-time logs

server ubuntu 20.04 LTS
MYSQL - 8
umami - latest
sever - nginx

```sh
yarn check-db
yarn update-db
service mysql restart
service nginx restart
```

![Screenshot from 2022-10-25 11-42-48](https://user-images.githubusercontent.com/10300271/197696272-8ec87f76-6eee-4a91-b475-1233b55ac55d.png)

**Comment by giuliomagnifico** [comment]
> ## 2) Copy/Paste
> If using docker-compose: `docker exec -it <umami_db_container_name> psql -U umami -W umami`

I get this error:

`root@analytics:/opt/umami# docker exec -it umami_umami_1 psql -U umami -W umami
OCI runtime exec failed: exec failed: unable to start container process: exec: "psql": executable file not found in $PATH: unknown`

**Comment by ddeimeke** [comment]
> > ## 2) Copy/Paste
> > If using docker-compose: `docker exec -it <umami_db_container_name> psql -U umami -W umami`
> 
> I get this error:
> 
> `root@analytics:/opt/umami# docker exec -it umami_umami_1 psql -U umami -W umami OCI runtime exec failed: exec failed: unable to start container process: exec: "psql": executable file not found in $PATH: unknown`

Maybe you do not use the DB container but the application container.

**Comment by briancao** [comment]
We had a lot of fixes to apply to auth and migrations in 1.39.2.  Should be fixed tonight.

**Comment by mikecao** [comment]
Fixed in v1.39.2.

**Comment by krautwigundrueben** [comment]
Hi guys, 

I updated from v1.38.0 to v1.39.5 and ran into a similar error:
- first, my postgres role didn't have su rights, so migration failed with `ERROR: permission denied to create extension "pgcrypto"`
- altered role to superuser
- now error says that column event_uuid doesn't exist in event table; it just has event_id, not _uuid

can you help me fix that and restore my data? I would have assumed the migration to be retried after fixing the rights issue and deploying again. it doesn't even say any more, that migrations haven't been applied, only `4 migrations found in prisma/migrations`. 

Thanks in advance!
Max
